spec/features/idv/in_person_threatmetrix_spec.rb

```diff
@@ -0,0 +1,309 @@
+require 'rails_helper'
+require 'action_account'
+require 'axe-rspec'
+
+RSpec.describe 'In Person Proofing Threatmetrix', js: true, allowed_extra_analytics: [:*] do
+  include InPersonHelper
+
+  let(:sp) { :oidc }
+  let(:user) { user_with_2fa }
+  let(:enrollment_code) do
+    JSON.parse(
+      UspsInPersonProofing::Mock::Fixtures.request_enroll_response,
+    )['enrollmentCode']
+  end
+  let(:stdout) { StringIO.new }
+  let(:stderr) { StringIO.new }
+  let(:argv) { ['review-pass', user.uuid, '--reason', 'INV1234'] }
+  let(:action_account) { ActionAccount.new(argv:, stdout:, stderr:) }
+  let(:review_pass) { ActionAccount::ReviewPass.new }
+  let(:review_reject) { ActionAccount::ReviewReject.new }
+  let(:include_missing) { true }
+  let(:config) { ScriptBase::Config.new(include_missing:, reason: 'INV1234') }
+  let(:enrollment) { InPersonEnrollment.last }
+  let(:profile) { enrollment.profile }
+
+  before do
+    allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+    allow(IdentityConfig.store).to receive(:in_person_proofing_enforce_tmx).and_return(true)
+    ServiceProvider.find_by(issuer: service_provider_issuer(sp)).
+      update(in_person_proofing_enabled: true)
+  end
+
+  def deactivate_profile_update_enrollment(status:)
+    # set fraud review to pending:
+    profile.deactivate_for_fraud_review
+    # update the enrollment:
+    enrollment.update(
+      status: status,
+      proofed_at: Time.zone.now,
+      status_check_completed_at: Time.zone.now,
+    )
+    profile.reload
+    enrollment.reload
+  end
+
+  shared_examples_for 'initially shows the user the barcode page' do
+    it 'shows the user the barcode page', allow_browser_log: true do
+      expect_in_person_step_indicator_current_step(
+        t('step_indicator.flows.idv.go_to_the_post_office'),
+      )
+      expect(page).to have_content(t('in_person_proofing.headings.barcode').tr(' ', ' '))
```

Comment:
*Not* for this PR, especially because this was just moved around (and I think I may have written the original line I'm now objecting to 😆), but I wonder if we can move the `.tr(' ', ' '))` pattern into a helper method explaining what the heck it's doing later on. It's not _at all_ intuitive that these are two different strings (replacing a non-breaking space with a regular space, or vice versa).

But again, *not* for now. I'll file a ticket.